h2o-core/src/test/java/hex/AUUCTest.java

```diff
@@ -53,6 +53,20 @@ public void testAUUCCompareCausalMl() {
         double resultAUUC = ArrayUtils.sum(causalMlGain)/(causalMlGain.length+1);
         assert auuc.auuc() - resultAUUC < 10e-10;
     }
+    
+    @Test
+    public void testAUUCNormalizedCompareCausalMl() {
+        double[] probs =  new double[]{0.1, -0.1, 0.2, 0.5, 0.55, 0.13, -0.2, 0.11, 0.3, 0.9};
+        double[] y =  new double[]     {0, 0, 0, 1, 1, 0, 0, 0, 1, 1};
+        double[] treatment =  new double[]{0, 0, 0, 0, 1, 1, 1, 1, 1, 1};
+        AUUC auuc = doAUUC(10, probs, y, treatment, AUUC.AUUCType.gain);
+
+        double[] causalMlGainNormalized = new double[]{0, 0, 0, 0, 1, 0.6, 0.28, 0.85333, 1.26, 1};
```

Comment:
These referential numbers we get from causalml library? Can we have quick demo in the comment how we can reproduce those numbers?

Reply:
Or have a test just in python, and leave it here as a smoke test with note that this can fail in case we change the inner implementation?

Reply:
I added code into the comment of the test class. To make pyunit, I should delegate AUUC calculation into API, and that is what I dont want now.